index.js

```diff
@@ -181,8 +181,9 @@ module.exports = (config) => {
 					const data = {
 						suite_id: suiteId,
 						name: runName,
-						include_all: true,
+						include_all: !config.plan.onlyRunIds,
 						config_ids,
+						case_ids: ids,
```

Comment:
do we always need `case_ids` here if `include_all` is `true`?

Reply:
we do not, but as far as I am concerned it does not change anything.

Reply:
could we update the code like adding `case_ids` only if `include_all` is false?

Reply:
yup, but as long as `include_all` is `true` the `case_ids` is not being considered:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/93591284/155540741-51101909-f5b3-4134-8a74-70a5e882d067.png">

Reply:
sounds good!